Kitodo/src/test/java/org/kitodo/forms/GeneratorSwitchTest.java

```diff
@@ -8,7 +8,8 @@
  * For the full copyright and license information, please read the
  * GPL3-License.txt file that was distributed with this source code.
  */
-package org.kitodo.util;
+
+package org.kitodo.forms;
```

Comment:
Merge conflict?

Reply:
No. As I wrote above, this class is part of a JSF form class. That's why I correctly moved them to the `forms` package. By the way, you have already reviewed this code. You do not need to review it again.

Reply:
Exactly I have reviewed this code that's why I know that this package assignment is wrong. Could you show me page to which responds this form? Because I don't see any.  Even more, I can say 100% that this whole stuff doesn't work. But as you don't want my review I will not elaborate what is wrong.

Reply:
But… like @Beacze says it has changed since https://github.com/kitodo/kitodo-production/pull/1819/files#diff-a287091c81593b5384e4e9f24aaccdd4 so it has to be reviewed and discussed again. I know the PRs need each other but switching a class to another package between PRs doesn't seem right.